pkg/elastic/elasticsearch.go

```diff
@@ -56,11 +57,100 @@ func NewElasticRepository(log *zap.Logger, config *configuration.ElasticsearchCo
 	return repository, nil
 }
 
+func (repository *ElasticRepository) FilterLogs(params logs.Parameters) ([]string, error) {
+
+	numParameters := 0
+	query := fmt.Sprintf(`{
+	"query": {
+	"bool": {
+		"should": [ `)
```

Comment:
Consider using `map[string]intefrace{}` instead of `query += fmt.Sprintf()`. Like  
```
query := map[string]interface{}{
    "query": map[string]interface{}{
      "match": map[string]interface{}{
        "title": "test",
      },
    },
  }
```
and then marshalling this with the JSON

Reply:
Thank you so much for the review @igor-karpukhin. I will make the required changes.